extensions/2.0/Vendor/GRIFFEL_bim_data/schema/GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,105 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "GRIFFEL_bim_data Serializable Object",
+    "type": "object",
+    "description": "Contains same properties as embedded version plus properties to node map. Used only when data is saved in a separate file.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
```

Comment:
The standalone schema does not have glTF's `extras` and `extensions` properties. So either this line should be removed, or those two properties added.

Reply:
The line is removed.